Jenkinsfile

```diff
@@ -603,7 +603,7 @@ pipeline {
                     }
```

Comment:
Needs suggestion: It looks to me that once we make MLIR the default build, the `test_conv_igemm_mlir_*_small` will be exercised by all smoke stages therefore we should get rid of the entire "Smoke MLIR" stage.

Reply:
Correct.

Reply:
Will get rid of the entire "Smoke MLIR"